library/Faktory/Worker.hs

```diff
@@ -86,87 +89,82 @@ untilM_ predicate action = do
         untilM_ predicate action
     )
 
--- | Creates a new faktory worker, @'action'@ is ran with @'WorkerConfig'@ before
--- polling begins. Jobs received are passed to @'handler'@. The worker's
--- connection is closed when job processing ends.
-withRunWorker ::
-  (HasCallStack, FromJSON args)
-  =>  Settings
+-- | Returns the @'WorkerId' associated with a @'Worker'.
+workerID :: Worker -> WorkerId
+workerID Worker{workerId} = workerId
```

Comment:
To be consistent with the style of this library, we should probably call this `workerId`.